src/gui/FileBrowser.cpp

```diff
@@ -1276,5 +1454,18 @@ QString FileItem::extension(const QString & file )
 	return QFileInfo( file ).suffix().toLower();
 }
 
+QString FileItem::defaultFilters()
+{
+	// TODO: Supported extensions should be in a centralized location
+	auto simpleExtensions
+		= QString{"*.mmp *.mpt *.mmpz *.xpf *.xml *.xiz *.sf2 *.sf3 *.pat *.mid *.midi *.rmi *.dll *.lv2"};
+#ifdef LMMS_BUILD_LINUX
+	simpleExtensions += " *.so";
+#endif
+	auto audioExtensions = QString{"*.wav *.ogg *.ds *.flac *.spx *.voc *.aif *.aiff *.au *.raw *.wav *.ogg *.ds "
+								   "*.flac *.spx *.voc *.aif *.aiff *.au *.raw"};
```

Comment:
mp3 just disappeard.  ;)

PS. why are some of the suffixes duplicated? Is this by design?

Reply:
Actually, it's not some of the suffixes. The string is duplicated so they all appear twice. Surely this is wrong? They show up fine where I've tested it though (afp).

Reply:
I should've waited for a review, but didn't know when that was going to happen and needed this ASAP. I’ll send in a fix immediately.

Reply:
Fixed in #6978.

Reply:
Great!